### ISSUE TITLE
When trying to update the VS Code Plugin v0.15.9 you get an error :

### ISSUE THREAD

**Issue by StephenWeatherford** [microsoft/vscode-azurearmtools#1572]
from @tshaiman

              when trying to update the VS Code Plugin v0.15.9 you get an error : 
```
2023-03-23 03:31:56.154 [info] [perf] Render performance baseline is 16ms
2023-03-23 03:32:14.432 [error] RequestService#request (browser) - error https://marketplace.visualstudio.com/_apis/public/gallery/extensionquery {"name":"Canceled"}
2023-03-23 03:32:22.643 [error] Signature: PackageIntegrityCheckFailed
    at yt.download (vscode-file://vscode-app/c:/Users/tshaiman/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/node/sharedProcess/sharedProcessMain.js:91:468)
    at async X.x (vscode-file://vscode-app/c:/Users/tshaiman/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/node/sharedProcess/sharedProcessMain.js:91:20760)
    at async X.h (vscode-file://vscode-app/c:/Users/tshaiman/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/node/sharedProcess/sharedProcessMain.js:91:18897)
2023-03-23 03:32:22.663 [error] PackageIntegrityCheckFailed: Signature: PackageIntegrityCheckFailed
    at yt.download (vscode-file://vscode-app/c:/Users/tshaiman/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/node/sharedProcess/sharedProcessMain.js:91:468)
    at async X.x (vscode-file://vscode-app/c:/Users/tshaiman/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/node/sharedProcess/sharedProcessMain.js:91:20760)
    at async X.h (vscode-file://vscode-app/c:/Users/tshaiman/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/node/sharedProcess/sharedProcessMain.js:91:18897)
```

_Originally posted by @tshaiman in https://github.com/microsoft/vscode-azurearmtools/issues/1557#issuecomment-1480461586_

**Comment by StephenWeatherford** [comment]
That's weird.  As I mentioned, I saw that yesterday, too, but was able to install it manually, and it looked like it probably was coming from another extension.

Telemetry indicates that over 1600 people are running the new version, so presumably installation wasn't an issue for them.  Several hundred of those are Mac/Linux (the machine it failed on for me yesterday was a Mac - my Windows machine didn't have any issues installing automatically today).

All that said, I was able to repro on Windows after upgrading vscode.  Trying to figure out what's going on.
